collections/ansible_collections/massopencloud/esi/roles/node/tasks/get_agent_names.yaml

```diff
@@ -0,0 +1,34 @@
+- name: Get agents
+  kubernetes.core.k8s_info:
+    kind: Agent
+    api_version: agent-install.openshift.io/v1beta1
+    namespace: "{{ manage_agents_namespace }}"
+  register: agents
```

Comment:
I feel like this task should be done in the cloudkit playbook, and passed into here.